Improve cannot find pdflatex error in Flatpak

hyperupcall opened the pull request:
This improves the error message for Flatpak users when pdflatex cannot be found. This makes the experience much more clear for the end user.

Closes #3996

hyperupcall committed:
ux: Improve no pdflatex program error in Flatpak

bhennion commented:
Hi, and thanks for the PR!

This looks good to me. I think this should go into the next bugfix release (1.1.2).
Would you mind rebasing this onto the release-1.1 branch? (I can help or even do it myself if you don't know how to)

hyperupcall committed:
fix: Adjust to use g_free functions

hyperupcall committed:
fix: Readd include

hyperupcall committed:
style: clang-format

hyperupcall commented:
Sure

I did the following:
- `git rebase --onto=release-1.1 master improved-flatpak-latex-ux`
- changed GitHub to merge into `xournalpp:release-1.1` instead of `xournalpp:mater`
- Fixed up some misc things since changing the rebase

Is this what you're looking for?

bhennion commented:
Yes! Thank you!

As this pretty innocuous, I'll be merging this in the release branch in 48 hours, unless an objection is raised.

hyperupcall commented:
Sounds good!

rolandlo commented:
The `xournalpp.pot` file should not be part of the PR, I think. It will be regenerated automatically after merging the PR.

hyperupcall commented:
@rolandlo Sounds good - I'll fix that later for my four other PRs too

bhennion committed:
remove .pot modifications

bhennion commented:
Thanks for the good work!
